inductiva/__init__.py

```diff
@@ -66,3 +66,31 @@ def _check_for_available_package_update():
 
 
 _check_for_available_package_update()
+
+
+def _supports_ansi():
+    """Checks if we support ansi formatting for colors and bolds"""
+    user_disable_ansi = utils.format_utils.getenv_bool("INDUCTIVA_DISABLE_ANSI",
+                                                       False)
+    if sys.platform.startswith("win"):
+        return "TERM" in os.environ and os.environ[
+            "TERM"] == "xterm" and not user_disable_ansi
+    return hasattr(sys.stdout,
+                   "isatty") and sys.stdout.isatty() and not user_disable_ansi
+
+
+_ansi_enabled = _supports_ansi
```

Comment:
this should be a function call:

```suggestion
_ansi_enabled = _supports_ansi()
```

so that the other modules can just use `_ansi_enabled` rather than having to call the `_supports_ansi` function multiple times.

Reply:
Ah, this was a typo